### ISSUE TITLE
Activity Page: GSC Update to include FOG data

### ISSUE THREAD

**Issue by patriciarussellCAS** [bcgov/cas-reporting#361]
#### Description:
Update to the General Stationary Combustion excluding line tracing page (#253 )
We need to add a datafield for fat, oil & grease (FOG)
See sheet 3 in the BCIERS Reporting Fields v2 spreadsheet (Step 3.1.3)

#### Acceptance Criteria:

Given that I am an Industry Operator with the primary NAICS code of 311614
When I am on the GSC excluding line tracing activity page
Then I need to fill out the following field: 
"Equivalent Emissions related to fat, oil and grease collection, refining and storage"

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by dleard** [comment]
Question for BA: 
If we're going to capture this data on the gsc activity page. Does this emission have a unit/fuel? Or is it a single emission that does not have those parent entities?

**Comment by patriciarussellCAS** [comment]
From SME teams channel: 
Jairo: "I think FOG emissions and Kiln emissions (steps 3.1.3 and 3.1.2 in the spreadsheet) should not be stored as emissions in the database. The main reason is that, in my mind, emission entities should be mutually exclusive; and FOG and Kiln emissions are subsets of emissions reported under GSC excluding line tracing. They could be stored as attributes, not sure of which entity."

Pierre: "Hmmm this is tricky. We built the system assuming the "has fuel" and "has unit" is a property of the activity, so having reporting items not following that pattern will require handling special cases, which is quite a bit of work.
I would suggest making these (3.1.2 - GSC items for cement or lime manufacturing) and (3.1.3 - Naics 311614) cases fall under their individual activities as additional fields, instead of adding it under GSC. Would that be acceptable?"

Robb: "yes, i think this should be fine"

**Comment by rdromey** [comment]
@dleard to write dev checklist

**Comment by dleard** [comment]
@patriciarussellCAS The details in this card are outdated. From my meeting notes, what we decided on is to allocate emissions to an unregulated FOG product (which we have created) and exclude those emissions in our compliance calculations (which the compliance service already does). Since the reporter can select FOG as a product & allocate emissions to it, we already handle the need to exclude those emissions as far as the compliance calculation service is concerned. 

We _are_ asking for production data for the FOG product in this case & we should ask IRC to review the FOG workflow & compliance summary results to test that it is working as intended. If they don't want us to collect production data for FOG as it is an unregulated product, we can filter it out of the production data page & ignore it when iterating over products in the compliance summary (the product data results in 0 since there are no compliance emissions for this product).

**tldr:** I don't think this ticket needed (it is complete) & we should create a new ticket if there are changes to the workflow required after a review from IRC

**Comment by dleard** [comment]
I've was thinking about this last night, and I think we should filter out unregulated products from the product page & compliance summary loop

**Comment by patriciarussellCAS** [comment]
Just flagging this bug ticket in case it is impacted by this at all: #543

**Comment by rdromey** [comment]
Will be completed as part of #543
